motif-docs/pages/guides/config/using-code.mdx

```diff
@@ -13,12 +13,21 @@ import dashBoardPic from '../../../../public/dashboard-maintenance-window.png'
 
 To use your own file, you'll need to create a compatible JSON file and add it to the path you indicated in the configuration.
 
-It can be useful to use our JSON schema for autocompletion: https://app.flightcontrol.dev/schema.json.
+## How do `flightcontrol.json` files fit into Flightcontrol concepts?
+- A git repo can have multiple `flightcontrol.json` files each of which corresponds to a single `Project`
+- A `Project` can have multiple `Environments`
+- An `Environment` can have multiple `Service`'s
+
+```mermaid
+graph TD;
+    flightcontrol.json-->Project;
+    Project-->Environment_A;
+   Project-->Environment_B;
+    Environment_A-->Service_1;
+   Environment_A-->Service_2;
+```
```

Comment:
Not sure if you checked the preview deployment, but it seems we don't currently support mermaid.

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/8813276/214091458-204b3c1b-04a1-4122-83bb-d80aa187db66.png">

I'll just close this PR for now until we add mermaid support